documentation/src/main/asciidoc/public/reference/index.adoc

```diff
@@ -0,0 +1,70 @@
+= Hibernate Search {hibernateSearchVersion}: Reference Documentation
+:doctype: book
+:revdate: {docdate}
+:sectanchors:
+:anchor:
+:toc: left
+:toclevels: 4
+:docinfodir: {docinfodir}
+:docinfo: shared,private
+:title-logo-image: image:hibernate_logo_a.png[align=left,pdfwidth=33%]
+
+:relfileprefix: ../../../just-to-be-removed/
```

Comment:
Those "just to be removed" things are confusing. I'd suggest adding a comment each time you do that, explaining why?

Reply:
added a comment with an example + turned out it wasn't needed in all the places -- so I've simplified some of these URLs.